pages/Documents/ConversationalAI/Third-PartyBots/bot-conversation-errors.md

```diff
@@ -176,6 +176,24 @@ Mostly credential issues.
 #### Solutions
 Check for your bot credentials.
 
+### com.liveperson.bot-connectors-worker.error.aivendor.dialogflowcx.parse-message.parser-error
+
+#### Description
+Testing connection with DialogFlow CX failed.
+#### Reasons
+Mostly credential issues.
+#### Solutions
+Check for your bot credentials.
+
+### com.liveperson.bot-connectors-worker.error.aivendor.dialogflowcx.parse-message.multiple-actions
+
+#### Description
+Error parsing multiple actions from DialogFlow CX.
+#### Reasons
+It was not possible to parse multiple actions from DialogFlow CX
+#### Solutions
+Check the bot response on DialogFlow CX side. 
```

Comment:
`Verify the bot response correspond to the standards we provide in our developer's documentation. `

Also link to the relevant part of documentation if we support it.